### PR TITLE
PT-translations in Basic Applications

### PR DESCRIPTION
**Pull request by fonsecagabriella**

### CODE REVIEW

File: i18n/pt/docusaurus-plugin-content-docs/current/basic_applications/introduction.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# 🟢 Introdução
+
+Agora que você aprendeu algumas técnicas básicas de engenharia de prompt, é hora de aprender como usá-las para resolver problemas simples do dia a dia. Esta seção aborda tarefas como escrever/resumir e-mails e criar contratos.
+
+Os exemplos apresentados nos artigos a seguir demonstram a variedade de aplicações para as quais você pode aplicar a engenharia de prompt.
+
+:::note
+Nesses exemplos, utilizamos tanto o ChatGPT quanto o GPT-3. Normalmente não importa qual você utiliza.
+
+
```
Comment (trigaten) [inline i18n/pt/docusaurus-plugin-content-docs/current/basic_applications/introduction.md:14]: Remove the spaces here

**Review by fonsecagabriella — COMMENTED** [review]
Spaces in the "Introduction" were removed.

File: i18n/pt/docusaurus-plugin-content-docs/current/basic_applications/blog_generation.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Escreva uma estrutura para uma postagem de blog curta sobre por que as Cactos de Natal são uma ótima compra.
+
+Existem muitos produtos que tentam fazer isso, mas você pode obter resultados consistentes fazendo você mesmo.
+
+
+```text
+Crie o esboço de um post de um post de blog curto sobre por que as Cactos de Natal são uma ótima compra.
+
+
```
Comment (trigaten) [inline i18n/pt/docusaurus-plugin-content-docs/current/basic_applications/blog_generation.md:23]: remove extra space

**Comment by trigaten** [comment]
Check how zh-Hans does images: https://github.com/trigaten/Learn_Prompting/tree/main/i18n/zh-Hans/docusaurus-plugin-content-docs/assets

The files should be named the same as the English version. Then, you can keep all the image paths exactly the same in your translated files and docusaurus will just use the pt version.

**Pull request closed without merging**